data/dfndata/items/skills/tools/alchemy.dfn

```diff
@@ -651,3 +725,67 @@ restock=20
 decay=1
 good=51
 }
+
+[fullvials]
+{
+get=0x185d 0x185e
+}
+
+[0x1810]
+{
+get=base_item
+name=Spinning Hourglass
+id=0x1810
+weight=100
+restock=20
+movable=1
+decay=1
+good=51
+}
+
+[0x1811]
+{
+get=base_item
+name=Spinning Hourglass
+id=0x1810
+weight=100
+restock=20
+movable=1
+decay=1
+good=51
+}
+
+[spinninghourglass]
+{
+get=0x1810 0x1811
+}
+
+[blowpipe]
+{
+get=base_item
+name=blowpipe
+id=0xE8A
+colour=0x3B9
+weight=100
+value=9 4
+restock=20
+decay=1
+good=51
+maxuses=75
+usesleft=25
+script=2200// uses left tooltip
+script=4031// craft tool
+}
+
+[glassblowingbook]
+{
+get=base_item
+name=Crafting glass with Glassblowing
+id=0xFF4
+weight=500
+movable=1
+pileable=1
+decay=1
+value=10637 5318
+script=5053
```

Comment:
Change script ID to 5054 - modified in jse_fileassociations.scp already to resolve conflict with some mining stuff.